docs/cmd/kn_service_create.md

```diff
@@ -47,8 +47,10 @@ kn service create NAME --image IMAGE [flags]
       --max-scale int            Maximal number of replicas.
       --min-scale int            Minimal number of replicas.
   -n, --namespace string         List the requested object(s) in given namespace.
+      --no-wait                  Don't wait for service to become ready.
```

Comment:
i would prefer if we change it to --wait=bool with a desired default.

Reply:
I change this. This `--no-wait` style was a reminiscence to GNU getopts where long boolean options are used to have `--with-` and `--no-` prefixes. But guess that's not so common anymore.

Reply:
@cppforlife @sixolet Just thought about the naming of the option again. If we make waiting the default value, then a `--wait=true` doesn't make much sense and doesn't provide any extra value. Also, having an option with a string parameter allows for additional errors like providing a non-standard value (like `no`, `off`, ....). So I'd really prefer a boolean option here.

My suggestion would be to fall back to the original suggestion of `--async`, or any other boolean toggle like `--no-wait`, `--background`, `--apply`, ...

Reply:
IMO, `--no-wait` reads simple and should be easy to use with default value of false.